manifests/s/SoftMaker/FreePDF/2022/3.0.8/SoftMaker.FreePDF.2022.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: SoftMaker.FreePDF.2022
+PackageVersion: 3.0.8
+InstallerType: inno

```

```suggestion
InstallerType: inno
AppsAndFeaturesEntries:
- DisplayVersion: '3.08'
```
The publisher markets the version as `3.0.8` but the version reported to Registry (visible in Control Panel) is `3.08`